fg/main.go

```diff
@@ -0,0 +1,202 @@
+package main
```

Comment:
⭐ _(nice to have)_

meta: can we add a `Makefile` or similar so that there's a `make build` command that runs this file?

Reply:
agreed